### PR TITLE
chore(contributing): update contributor guidelines

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

**Review by mtomcal — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,145 +1,125 @@
...
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
+    ```
 
-The *type* should always be lowercase as shown below.
+4. You can now run any of this scripts from the root folder.
```
Comment (anescobar1991) [inline CONTRIBUTING.md]: `this` should be `these`.

**Review by anescobar1991 — CHANGES_REQUESTED** [review]

**Review by 10xLaCroixDrinker — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,145 +1,125 @@
...
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
    $ npm install
```

**Review by 10xLaCroixDrinker — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,145 +1,125 @@
...
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
 
-```bash
-fix(home/transactions): txn actions not working
-```
+## Submitting a new feature
 
-This header makes it clear that the fix is limited in scope to transactions within the home route/context.
+When submitting a new feature request or enhancement of an existing feature please review the following:-
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
When submitting a new feature request or enhancement of an existing feature please review the following:
```

**Review by 10xLaCroixDrinker — CHANGES_REQUESTED** [review]

**Review by 10xLaCroixDrinker — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,145 +1,125 @@
...
+Please include a snippets of the code of the new feature.
 
-```bash
-$ git commit -m "fix(nav): incorrect URL for Travel nav item :P"
-```
+### Describe the suggested enhancement
 
-> Note that the _Bad_ example results in a longer commit header. This is partly attributed to the scope not being more specific and personal expression tacked on the end.
+A clear and concise description of the enhancement to be added include a step-by-step guide if applicable.
+Add any other context or screenshots or animated GIFs about the feature request
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
Add any other context or screenshots or animated GIFs about the feature request.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,145 +1,125 @@
...
 
-```bash
-$ git commit -m "fix(nav): broken URL for Travel"
-```
+A clear and concise description of what the bug is.
 
+### Provide step-by-step guide on how to reproduce the bug
 
-#### Body and Footer (optional)
+Steps to reproduce the behavior, please provide code snippets or a link to repository
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
Please provide code snippets or a link to repository with steps to reproduce the behavior.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,145 +1,125 @@
...
 
-The body and footer should wrap at 80 characters.
+### Expected behavior
 
-The **body** describes the commit in more detail and should not be more than 1 paragraph (3-5 sentences). Details are important, but too much verbosity can inhibit understanding and productivity -- keep it clear and concise.
+Please provide a description of the expected behavior
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
Please provide a description of the expected behavior.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,145 +1,125 @@
...
+- Join our [Slack channel](https://one-amex.slack.com) request an invite [here](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjY)
 
-NavItem now supports an "isActive" property. This property is used to control the styling of active navigation links.
+## Coding conventions
+
+### Git Commit Guidelines
 
-Closes #21
-```
+We follow [conventional commits](https://www.conventionalcommits.org/) for git commit message formatting. These rules make it easier to review commit logs and improve contextual understanding of code changes. This also allows us to auto-generate the CHANGELOG from commit messages.
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]: Breaking line length:

```suggestion
We follow [conventional commits](https://www.conventionalcommits.org/)
for git commit message formatting. These rules make it easier to review
commit logs and improve contextual understanding of code changes.
This also allows us to auto-generate the `CHANGELOG` from commit messages.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,145 +1,125 @@
...
 
-We follow precise rules for git commit message formatting. These rules make it easier to review commit logs and improve contextual understanding of code changes. This also allows us to auto-generate the CHANGELOG from commit messages.
+## Developing
 
-Each commit message consists of a **header**, **body** and **footer**.
+### Installation
 
-#### Header
+1. Fork the repository `iguazu-rpc` to your GitHub account.
+2. Afterwards run the following commands in your terminal
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]: The other pointers are using grammatical breaks (`.`, eg), would make sense to do that with all the points.

```suggestion
2. Afterwards run the following commands in your terminal:
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,145 +1,125 @@
...
+    $ cd iguazu-rpc
+    ```
 
-Syntax:
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
3. Install the dependencies by running:
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,145 +1,125 @@
...
 
-#### Type
+    ```bash
+    $  npm install
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
    $ npm install
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,145 +1,125 @@
...
-Another scenario for scope is using a `route/scope` (or `context/scope`) syntax. This would be useful when a commit only affects a particular instance of code that is used in multiple places.
+Runs unit tests **and** verifies the format of all commit messages on the current branch.
 
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
+- **`npm posttest`**
 
-```bash
-fix(home/transactions): txn actions not working
-```
+Runs linting on the current branch, checks that the commits follow [conventional commits](https://www.conventionalcommits.org/) and verifies that the `package-lock.json` file includes public NPM registry URLs.
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
Runs linting on the current branch, checks that the commits follow
[conventional commits](https://www.conventionalcommits.org/) and
verifies that the `package-lock.json` file includes public NPM registry URLs.
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -1,145 +1,125 @@
...
+- Browser (if applies) [e.g. chrome, safari]
+- Version of iguazu-rpc: [e.g. 5.0.0]
+- Node version:[e.g 10.15.1]
 
-```bash
-Closes #17, #26
-Related #37
-```
-> Issues should always be referenced before pull requests, as shown above.
+### Security Bugs
```
Comment (Francois-Esquire) [inline CONTRIBUTING.md]:
```suggestion
### Security bugs
```

**Review by Francois-Esquire — CHANGES_REQUESTED** [review]

**Comment by Francois-Esquire** [comment]
Added a few suggestions, mostly grammatical and for staying consistent within the document. Otherwise looks solid.

**Comment by nellyk** [comment]
Thank you for the detailed review 

> Added a few suggestions, mostly grammatical and for staying consistent within the document. Otherwise looks solid.

**Review by Francois-Esquire — APPROVED** [review]

**Review by mtomcal — APPROVED** [review]

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by anescobar1991 — APPROVED** [review]

**Pull request merged**